docs/source/using/managing-minishift.adoc

```diff
@@ -293,23 +293,32 @@ CONTAINER    IMAGE                   COMMAND                CREATED        STATU
 71fe8ff16548 openshift/origin:v1.5.1 "/usr/bin/openshift s" 4 minutes ago  Up 4 minutes  origin
 ----
 
-[[experimental-functionality]]
-== Experimental functionality
+[[experimental-features]]
+== Experimental Features
 
-To allow users and developers to get early access to some upcoming features, we offer a way to enable experimental functionality.
+If you want to get early access to some upcoming features and experiment, you can enable some of those in Minishift.
 
-You do this by setting the environment variable `MINISHIFT_ENABLE_EXPERIMENTAL` which will make additional flags available:
+[IMPORTANT]
+====
+Experimental features are not officially supported, and might break or result in unexpected behavior.
+To share your feedback on these features, you are welcome to link:https://github.com/minishift/minishift#community[contact the Minishift community].
+====
+
+You do this by setting the environment variable `MINISHIFT_ENABLE_EXPERIMENTAL`, which makes additional flags available:
 
 ----
 $ export MINISHIFT_ENABLE_EXPERIMENTAL=y
 ----
 
-At the moment this is only relevant for the xref:../command-ref/minishift_start.adoc#[`minishift start`] command where you get additional options related to  xref:../openshift/openshift-client-binary.adoc#[`oc cluster up`].
-By setting the environment variable, you get access to the `service-catalog` flag which allows you to provision OpenShift's link:https://docs.openshift.org/latest/architecture/service_catalog/index.html[service catalog].
+[[enabling-experimental-oc-flags]]
+=== Enabling experimental `oc cluster up` flags in `minishift start`
+
+By default, Minishift does not expose all xref:../openshift/openshift-client-binary.adoc#[`oc cluster up`] flags in the CLI, either because certain features are not considered stable or because Minishift does not yet support the latest options in `oc cluster up`.
+
+You can set the environment variable to enable the following options for the xref:../command-ref/minishift_start.adoc#[`minishift start`] command:
 
-You also get access to the `extra-clusterup-flags` flag, allowing you to pass any extra flags to `oc cluster up`.
-Minishift does per default not expose all `oc cluster up` flags in its CLI, either because certain features are not considered stable or simply because Minishift has not caught up yet with the latest options in `oc cluster up`.
-In these cases `extra-clusterup-flags` can be used to run experiments with these options.
+`service-catalog`::
+Enables provisioning the OpenShift link:https://docs.openshift.org/latest/architecture/service_catalog/index.html[service catalog].
 
-Be aware that these features can break or give a result you weren't expecting.
-We welcome your feedback when using them.
+`extra-clusterup-flags`::
+Enables passing any extra flags to `oc cluster up`.
```

Comment:
Enabling the ability to pass along any `oc cluster up` flag which is otherwise not directly exposed by the Minishift flags.